app/controllers/cart_items_controller.rb

```diff
@@ -47,32 +49,6 @@ def set_cart_item
   end
 
   def cart_item_params
-    params.require(:cart_item).permit(:quantity)
+    params.require(:cart_item).permit(:product_id, :quantity, :price, :user_id)
```

Comment:
does cart_item require user_id attribute here? we already have the cart linked to user. right?